packages/core/src/List/List.tsx

```diff
@@ -3,6 +3,24 @@ import { fontSize, space, width, compose } from 'styled-system'
 import themeGet from '@styled-system/theme-get'
 import { getPaletteColor } from '../utils'
 
+const tabSpacingSize = {
+  default: '40px',
+  md: '32px',
+  sm: '24px',
+  xsm: '16px',
```

Comment:
nb: our other sizes go from smallest to biggest

Reply:
{
 xsm: '16px',
 sm: '24px'
 md: '32px',
 default: '40px'
 }
 
 like this.?

Reply:
yes exactly 😄

Reply:
i updated it.